### DISCUSSION TITLE
How to disable `button and ActionButton` transform feedback on button:active or on button clicked properly

### DISCUSSION THREAD

**Discussion by yohanesray** [mantinedev/mantine#4059] [Q&A]
If we press button in mantine component. It will have transform feedback of `translateY(0.0625rem)`

https://user-images.githubusercontent.com/60679808/231471758-f1e6b0be-6725-4669-b042-a2a9ae66c25d.mov

how can i disable this properly?

What i have tried

1.  disabling it globally. and I need to add `!important` on it

```css
button {
  transform: none !important;
}
```

also if I do this. I can't work with custom transform anymore since the global styles will just override whatever transform value has

2. This one is working as expected, but when i need it to work with `ActionButton` as well

```tsx
<Button styles={{ root: { ':active': { transform: 'none' } } }}>
      hello world
</Button>
```

3. This one wouldn't work

```tsx
<ActionIcon styles={{ root: { button: { transform: 'none' } } }}>
    Something
  </ActionIcon>
```

**Comment by disillusionedowl** [comment]
```js
<ActionIcon sx={ { '&:active': { transform: 'none' } } }>🚀</ActionIcon>
```

see: https://mantine.dev/styles/sx/

Or change it globally in `theme`:

https://github.com/mantinedev/mantine/blob/master/src/mantine-styles/src/theme/default-theme.ts#L92

**Comment by rtivital** [comment]
https://mantine.dev/theming/theme-object/#activestyles
